### ISSUE TITLE
Recurrence.Builder exception

### ISSUE THREAD

**Issue by marcoRS** [mangstadt/biweekly#68]
On version 0.6.1 if you create a Recurrence Builder from a another Recurrence rule and try to add an xRule you get an UnsupportedOperationException exception. 

 ```java
Recurrence.Builder builder = new Recurrence.Builder(previousRecurrence);
builder.xrule("EXDATE", date.toString());
```
Running the above code results in the following exception. 

java.lang.UnsupportedOperationException at java.util.Collections$UnmodifiableMap.put(Collections.java:1441)  at biweekly.util.ListMultimap.put(ListMultimap.java:125) at biweekly.util.Recurrence$Builder.xrule(Recurrence.java:634)

This exception does not happen when creating a builder using a Frequency.

**Comment by mangstadt** [comment]
Thanks for the report!  I'll look into it.

By the way, it doesn't look like you're doing what you're trying to do correctly (if that makes sense) :confused: .  EXDATE is a completely separate property.  What you probably want to do is add an ExceptionDates property to the VEvent component.

**Comment by marcoRS** [comment]
Thanks for the advice unfortunately the EXDATE is being stored that way due to an api we are using that expects it in the RRule. But we do plan on extracting the EXDATES and using them in a VEvent.

**Comment by mangstadt** [comment]
Ah, got it.

Fixed in f335f473c34e641b06f116523fa54a518163a14e

Unfortunately, until a new stable version is released, the only work around is to not use the `Recurrence.Builder(Recurrence)` constructor.

Thanks again for the report! :grin:
